static/js/caching-config.js

```diff
@@ -20,7 +20,7 @@ const OTHER_API_PATHS_TO_CACHE_AS_CONTENT = ['/resources/:ID/thumbnail', '/bible
 
 const API_PATHS_TO_SKIP_CACHING = ['/resources/batch/metadata', '/resources/batch/content/text'];
 
-const CDN_URLS = ['https://cdn.aquifer.bible'];
+const CDN_URLS = ['https://cdn.aquifer.bible', 'https://aquiferstorage.blob.core.windows.net'];
```

Comment:
Oops this is my bad, I forgot that we were using the cdn.aquifer.bible. I will go ahead and rename all the data so that it uses that one and you can get rid of the the blob.core.windows one here.